Vagrantfile

```diff
@@ -0,0 +1,39 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+VAGRANTFILE_API_VERSION = '2'
+
+Vagrant.require_version '>= 1.5.0'
+
+Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
+  config.vm.hostname = 'instance-image'
+  if Vagrant.has_plugin?('vagrant-omnibus')
+    config.omnibus.chef_version = 'latest'
+  end
+  if Vagrant.has_plugin?('vagrant-berkshelf')
```

Comment:
I thought so too, but it seems to have been updated enough to work for this. I didn't have any problem deploying it. It even works with chefdk sanely. I'd like to keep it unless we run into issues. The only issue I ran into was fixing my ``$PATH`` to have the /opt/chef/bin path first.